tests/default.nix

```diff
@@ -42,24 +42,37 @@ let
   };
 in
 # We attempt to build & execute all configurations
-lib.pipe (testFiles ++ [ exampleFiles ]) [
-  (builtins.map (
-    {
-      name,
-      file,
-      cases,
-    }:
-    {
-      inherit name;
-      path = pkgs.linkFarm name (builtins.mapAttrs (moduleToTest file) cases);
-    }
-  ))
-  (helpers.groupListBySize 10)
-  (lib.imap1 (
-    i: group: rec {
-      name = "test-${toString i}";
-      value = pkgs.linkFarm name group;
-    }
-  ))
-  builtins.listToAttrs
-]
+(lib.pipe
+  (
+    testFiles
+    ++ [
+      exampleFiles
+    ]
+  )
+  [
+    (builtins.map (
+      {
+        name,
+        file,
+        cases,
+      }:
+      {
+        inherit name;
+        path = pkgs.linkFarm name (builtins.mapAttrs (moduleToTest file) cases);
+      }
+    ))
+    (helpers.groupListBySize 10)
+    (lib.imap1 (
+      i: group: rec {
+        name = "test-${toString i}";
+        value = pkgs.linkFarm name group;
+      }
+    ))
+    builtins.listToAttrs
+  ]
+)
```

Comment:
Hard to tell on mobile, but is this just a formatting change brought on by adding (unnecessary) parentheses?

Reply:
Yeah I think. I added them at some point when trying something else